zetaclient/chains/ton/observer/inbound.go

```diff
@@ -23,58 +20,13 @@ const (
 	// TODO: move to config
 	// https://github.com/zeta-chain/node/issues/3086
 	maxTransactionsPerTick = 100
-	// zero log sample rate for sampled logger (to avoid spamming logs)
-	logSampleRate = 10
 )
 
-// watchInbound watches for new txs to Gateway's account.
-func (ob *Observer) watchInbound(ctx context.Context) error {
-	return ob.inboundTicker(ctx, "WatchInbound", ob.observeGateway)
-}
-
-func (ob *Observer) watchInboundTracker(ctx context.Context) error {
-	return ob.inboundTicker(ctx, "WatchInboundTracker", ob.processInboundTrackers)
-}
-
-func (ob *Observer) inboundTicker(ctx context.Context, taskName string, taskFunc func(context.Context) error) error {
-	app, err := zctx.FromContext(ctx)
-	if err != nil {
-		return err
-	}
-
-	initialInterval := ticker.DurationFromUint64Seconds(ob.ChainParams().InboundTicker)
-	sampledLogger := ob.Logger().Inbound.Sample(&zerolog.BasicSampler{N: logSampleRate})
-
-	task := func(ctx context.Context, t *ticker.Ticker) error {
-		if !app.IsInboundObservationEnabled() {
-			sampledLogger.Info().Msgf("%s: inbound observation is disabled", taskName)
-			return nil
-		}
-
-		if err := taskFunc(ctx); err != nil {
-			ob.Logger().Inbound.Err(err).Msgf("%s failed", taskName)
-		}
-
-		newInterval := ticker.DurationFromUint64Seconds(ob.ChainParams().InboundTicker)
-		t.SetInterval(newInterval)
-
-		return nil
-	}
-
-	return ticker.Run(
-		ctx,
-		initialInterval,
-		task,
-		ticker.WithStopChan(ob.StopChannel()),
-		ticker.WithLogger(ob.Logger().Inbound, taskName),
-	)
-}
-
-// observeGateway observes Gateway's account for new transactions.
-// Due to TON architecture we have to scan for all net-new transactions.
+// ObserveGateway observes Gateway's account for new transactions.
```

Comment:
I think `ObserveGateway` could be `ObserveInbound`, for more consistent naming with `ObserverOutbound`